recipe/meta.yaml

```diff
@@ -37,125 +41,96 @@ requirements:
     - coreutils  # [osx]
     - patchelf  # [linux]
     - unzip
-    - nodejs
+    - nodejs >=18
     - {{ cdt('mesa-libgl-devel') }}  # [linux]
     - {{ cdt('mesa-dri-drivers') }}  # [linux]
     - {{ cdt('libselinux') }}  # [linux]
     - {{ cdt('libxdamage') }}  # [linux]
     - {{ cdt('libxxf86vm') }}  # [linux]
     - {{ cdt('libxext') }}     # [linux]
-
-outputs:
-  - name: rstudio-desktop
```

Comment:
Please keep the outputs. Once we get `libpam` as a CDT we can add `rstudio-server` here.